web/rootfs/defaults/meet.conf

```diff
@@ -37,6 +39,22 @@ location = /http-bind {
     proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
 }
 
+{{ if $ENABLE_XMPP_WEBSOCKET }}
+# xmpp websockets
+location = /xmpp-websocket {
+    proxy_pass {{ .Env.XMPP_BOSH_URL_BASE }}/xmpp-websocket;
+    proxy_http_version 1.1;
+
+    proxy_set_header Connection "upgrade";
+    proxy_set_header Upgrade $http_upgrade;
+
+    proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
+    proxy_set_header X-Forwarded-For $remote_addr;
+    proxy_read_timeout 3600s;
```

Comment:
Thiss value is too high, WS sends pings, so I. thinkwe can. do with the sssuggested 900, right?

Reply:
I misunderstand the documentation. I thought the default value of 890 was bumped.

Reply:
```suggestion
    proxy_read_timeout 900s;
```